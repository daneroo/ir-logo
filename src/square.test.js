// Link.react.test.js
import React from 'react'
import Square from './square'
import renderer from 'react-test-renderer'

test('Render a square', () => {
  const component = renderer.create(
    <Square />
  )
  let tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
