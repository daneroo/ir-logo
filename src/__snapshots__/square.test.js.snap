// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render a square 1`] = `
<polygon
  points="-100,100, 100,100, 100,-100, -100,-100"
  style={Object {}}
  transform=""
/>
`;
